workers/unity/Packages/com.improbable.gdk.mobile/Worker/MobileWorkerConnector.cs

```diff
@@ -50,6 +50,7 @@ protected override LocatorConfig GetLocatorConfig()
 
         protected override AlphaLocatorConfig GetAlphaLocatorConfig(string workerType)
         {
+            LoadDevAuthToken();
```

Comment:
It feels strange to _need_ to call this but it doesn't return anything. I'd rather it does something like: 

```
var token = LoadDevAuthToken();
var pit = GetDevelopmentPlayerIdentitityToken(token, ...);
...
```

Also we should actually throw an error/exception if the asset _doesn't_ exist which is an invariant of the method.

---

If a user wants to load the dev auth token another way, they can simply write their own method and call that instead.

Reply:
Especially as we have the pattern of 

```
var thing = GetThing();
```

for the rest of the flow, these could equally be class variables, but (rightly) aren't